engine/src/main/java/org/hibernate/search/engine/metadata/impl/AnnotationMetadataProvider.java

```diff
@@ -946,6 +991,22 @@ private void bindFieldAnnotation(
 					.indexNullAs( nullToken );
 		}
 
+		if ( !facetAnnotations.isEmpty() ) {
```

Comment:
true, true - that was something from before I did some refactoring. Removing it.